Popup/Popup.module.less

```diff
@@ -50,9 +87,30 @@
 		border-color: @agate-popup-border-color;
 		border-radius: @agate-popup-border-radius;
 
+		&.bottom,
+		&.fullscreen,
+		&.top {
+			max-width: 100%;
+		}
+
+		&.bottom {
+			border-radius: @agate-popup-border-radius @agate-popup-border-radius @agate-popup-fullscreen-border-radius @agate-popup-fullscreen-border-radius;
+		}
+
+		&.left {
+			border-radius: @agate-popup-fullscreen-border-radius @agate-popup-border-radius @agate-popup-border-radius @agate-popup-fullscreen-border-radius;
+		}
+
+		&.fullscreen {
+			border-radius: @agate-popup-fullscreen-border-radius;
+		}
+
+		&.right {
+			border-radius: @agate-popup-border-radius @agate-popup-fullscreen-border-radius @agate-popup-fullscreen-border-radius @agate-popup-border-radius;
+		}
+
 		&.top {
-			border-radius: @agate-popup-top-border-radius;
-			max-width: @agate-popup-top-max-width;
+			border-radius: @agate-popup-fullscreen-border-radius @agate-popup-fullscreen-border-radius @agate-popup-border-radius @agate-popup-border-radius;

```

Comment:
can you move these to their own variables? example: "agate-popup-right-border-radius"

Reply:
Yes, sure. Thank you!